src/test/java/uk/gov/hmcts/reform/demo/controllers/GetWelcomeTest.java

```diff
@@ -17,7 +17,7 @@
 public class GetWelcomeTest {
 
     @Autowired
-    private MockMvc mockMvc;
+    private transient MockMvc mockMvc;
```

Comment:
Is `transient` required here?

Reply:
fixes error prone [rule](https://pmd.github.io/pmd-6.3.0/pmd_rules_java_errorprone.html#beanmembersshouldserialize)

Reply:
👍